compiler/utils/debugutils.nim

```diff
@@ -493,7 +495,7 @@ template addInNimDebugUtilsAux(conf: ConfigRef; prcname: string;
         if indentLevel != 0: # print a delta stack
           # try to print only the part of the stacktrace since the last time,
           # this is done by looking for any previous calls in `debugUtilsStack`
-          {.line.}: # stops the template showing up in the StackTraceEntries
+          {.line:instantiationInfo(-2, true).}: # stops the template showing up in the StackTraceEntries
```

Comment:
```suggestion
          {.line: instLoc(-3).}: # stops the template showing up in the StackTraceEntries
```
A soft suggestion -- it's a bit shorter and also respects the `excessiveStackTrace` option. The depth should be correct, but I haven't tried it.